roles/openshift_provisioners/defaults/openshift-enterprise.yml

```diff
@@ -0,0 +1,3 @@
+---
+__openshift_provisioners_image_prefix: "registry.access.redhat.com/openshift3/ose-"
```

Comment:
I'm not a fan of the double underscore variables.  We were using l_x (lowercase L) prefix for variables we don't want users to muck with.  I realize some roles do that, but it's not my preference.

Reply:
Fixed, using dicts now